supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -5,5 +5,117 @@
 [[release-notes]]
 == Release notes
 
+[[release-notes-doc-texts]]
+=== Conventions for phrasing Release Notes doc texts
+
+A doc text is a short description of an engineering Bugzilla or Jira issue that is published in product Errata Advisories and Release Notes. Engineering supplies draft content, which is a summary of the issue, then technical writers edit or rewrite the draft content in accordance with _The IBM Style Guide_. You can write doc texts in more than one way, depending on the issue type: 
+
+* Enhancements
+* Bug Fixes
+* Known Issues
+* Technology Preview
+* Deprecated Functionality
+
+Use the following general structure for consistency.
+
+.Enhancement
+ 
+Use present tense in one of the following formats:
+
+----
+This enhancement <present tense explanation>.
+----
+
+----
+With this update, <present tense explanation>.
+----
+
+Example doc texts:
+----
+“This enhancement optimizes migration of an RBD volume from one Cinder back end to another when the volume resides within the same Ceph cluster. If both volumes are in the same Ceph cluster, Ceph performs data migration instead of the cinder-volume process. This reduces migration time.”
+----
+
+----
+"With this update, you can create application credentials to use keystone to authenticate applications."
+----
+
+.Bug Fixes
+
+Use past tense for the problem and present tense for the solution, in the following format:
+
+----
+Before this update,  <X problem> caused <Y situation> (OPTIONAL: under the following <Z conditions>). With this update, <fix> resolves the issue (OPTIONAL: and <agent> can <perform operation> successfully).
+----
+
+Example doc text:
+----
+“Before this update, the loopback device for Cinder iSCSI/LVM backend was not recreated after a system restart, which prevented the cinder-volume service from restarting. With this update, a systemd service recreates the loopback device and the Cinder iSCSI/LVM backend persists after a restart.”
+----
+
+.Known Issues
+
+Use present tense for the issue and the imperative form for the workaround in the following format:
+
+----
+There is currently a known issue <present tense explanation> under <X conditions>. 
+
+Workaround: <workaround>.
+----
+
+Example doc text:
```

Comment:
For all of these "Example doc texts" that precede, I'd suggest making them actual titles to the code block

```
.Example doc text
----
asdf
----
```